python/ray/autoscaler/command_runner.py

```diff
@@ -256,14 +277,27 @@ def _get_node_ip(self):
         else:
             return self.provider.external_ip(self.node_id)
 
-    def _wait_for_ip(self, deadline):
-        while time.time() < deadline and \
-                not self.provider.is_terminated(self.node_id):
-            logger.info(self.log_prefix + "Waiting for IP...")
-            ip = self._get_node_ip()
-            if ip is not None:
-                return ip
-            time.sleep(10)
+    def wait_for_ip(self, deadline):
+        # if we have IP do not print waiting info
+        # ip = self.get_node_ip()
+        # if ip is not None:
+        #     cli_logger.labeled_value("Fetched IP", ip)
+        #     return ip
```

Comment:
remove?

Reply:
lol this actually just works now for whatever reason, uncommented and going to keep it